src/kontrol/foundry.py

```diff
@@ -1301,3 +1302,28 @@ def foundry_node_printer(
     if type(proof) is APRProof:
         return FoundryAPRNodePrinter(foundry, contract_name, proof, omit_unstable_output=omit_unstable_output)
     raise ValueError(f'Cannot build NodePrinter for proof type: {type(proof)}')
+
+
+def init_project(skip_forge: bool, project_root: Path) -> None:
```

Comment:
```suggestion
def init_project(project_root: Path, *, skip_forge: bool) -> None:
```

Reply:
Done in https://github.com/runtimeverification/kontrol/pull/507/commits/ebaa18e2adf5f1d10560cdc17a94e1071e96d817